Package.swift

```diff
@@ -6,7 +6,7 @@ import PackageDescription
 let package = Package(
     name: "YipYipSwift",
     platforms: [
-        .iOS(.v10)
+        .iOS(.v12)
```

Comment:
Zet hier het minimum maar op 15. Dit is een interne dependency en we zullen geen apps meer ondersteunen met een minimale os versie onder 15.